src/hrc721.spec.ts

```diff
@@ -47,15 +47,15 @@ describe('HarmonyProvider', () => {
   })
 
   describe('ownerOf', () => {
-    it('should returns the owner of the tokenId token.', async () => {
+    it('should returns the owner of the tokenId token', async () => {
```

Comment:
nit: `should return` instead of "returns", same in other tests below

Reply:
Right, fixed!